npm/vue/cypress/support/index.js

```diff
@@ -1,3 +1,3 @@
 // created automatically when cy:open starts
-
+import 'regenerator-runtime/runtime'
```

Comment:
Why do we need this? Can we just update tsconfig to target `esnext`

Reply:
You are 💯 right we should get rid of babel in there...

Reply:
It's just that... webpack 😇

Reply:
OK it's been removed now. since it was unnecessary.